app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
+                        NotWearingMaskError)
+

```

Comment:
There should be a blank line between the import statements and the class definition to follow the code style guidelines.